src/order/services/order.service.ts

```diff
@@ -1,39 +1,50 @@
 import { Injectable } from '@nestjs/common';
-import { v4 } from 'uuid';
-
+import { randomUUID } from 'crypto';
```

Comment:
```suggestion
import { randomUUID } from 'node:crypto';
```

minor but I find it's more convenient and readable to prefix node imports with `node:`

Reply:
Done